src/PostgREST/DbStructure.hs

```diff
@@ -656,7 +656,7 @@ allViewsKeyDependencies =
       pks_fks as (
         -- pk + fk referencing col
         select
-          contype,
+          cast(contype as text) contype,
```

Comment:
```suggestion
          contype::text,
```

That's the style we use around there, I think.

Reply:
Yes, we use this style more (`::`), can confirm that tests run successfully for docker `postgres 15beta3` btw.

Reply:
@yevon Can you make this change? Somehow we cannot commit the suggestion.